ministryofjustice/correspondence_tool_staff
# CT-3113 Split prison number in method (to show only one) and apply in letter templates

## Description
Multiple prison numbers on cases need to show up on letter templates as only the first one in the list. This has been split by Comma and Spaces in between. (Commas converted to spaces and then split), then returning only the first one.

## Self-review checklist


### Screenshots
![image](https://user-images.githubusercontent.com/22935203/95459348-6f656a00-096b-11eb-96ed-015390d2dc0a.png)
![image](https://user-images.githubusercontent.com/22935203/95459481-a3d92600-096b-11eb-8af4-3b44bc138946.png)
![image](https://user-images.githubusercontent.com/22935203/95459513-ae93bb00-096b-11eb-8f7e-18941d3c86f0.png)
![image](https://user-images.githubusercontent.com/22935203/95459541-b81d2300-096b-11eb-85aa-6f6e3a13e2cb.png)

=== Another CASE, DISPATCH LETTERS ===


![image](https://user-images.githubusercontent.com/22935203/95459257-50ff6e80-096b-11eb-8495-ce239ca3860b.png)
![image](https://user-images.githubusercontent.com/22935203/95459640-daaf3c00-096b-11eb-9ae4-d6d86784a197.png)
![image](https://user-images.githubusercontent.com/22935203/95459707-eb5fb200-096b-11eb-921e-4c3bea904c2c.png)
![image](https://user-images.githubusercontent.com/22935203/95459740-f61a4700-096b-11eb-94df-e8f0ef3d72f6.png)
![image](https://user-images.githubusercontent.com/22935203/95459764-ffa3af00-096b-11eb-9950-9f4ed9b4b956.png)
![image](https://user-images.githubusercontent.com/22935203/95459804-092d1700-096c-11eb-8422-cc00392d61b9.png)
Word example:
![image](https://user-images.githubusercontent.com/22935203/95459897-219d3180-096c-11eb-9ee8-c470ab693564.png)


### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-3113

### Deployment
Need to run letter template seeder for each deployment manually

### Manual testing instructions
rake db:seed:dev:letter_templates
Create cases with multiple prison numbers and different address / recipient settings.
Go to each letter template
Ensure that wherever there is a Prison number, it only shows the first one
